vagrant/guest/bin/vhosts.sh

```diff
@@ -136,16 +141,46 @@ function generate_config {
     fi
 }
 
+function generate_varnish_config {
+    local service="varnish"
+    local site_name="$1"
+    local site_hosts="$2"
+    local site_path="$3"
+
+    local conf_dir="/etc/$service/sites.d"
+    local conf_file="$conf_dir/$site_name.vcl"
+    local conf_src=
+
+    local override="$site_path/.$service.vcl"
+    local status=
+
+    local site_pub=$(ls -1dU "$site_path"/{pub,html,htdocs} 2>/dev/null | head -n1)
+    [[ -n $site_pub ]] && site_pub=$(basename "$site_pub") || site_pub=pub
+
+    # figure out what to src the config from
+    if [[ -f "$override" ]]; then
+        # if override has not been copied or is different, we process it
+        if [[ ! -f "$conf_file" ]] || ! cmp "$override" "$conf_file" > /dev/null; then
+            status="(override)"
+            [[ -f "$conf_file" ]] && status="$status (updated)"
+
+            msg "   + $service config $status"
+            cp "$override" "$conf_file"
+        fi
+    fi
+}
+
 function process_site {
     local site_name="$1"
     local site_path="$2"
     local site_hosts[0]=
 
     # parse in list of custom hostnames if present
-    if [[ -f $site_path/.hostnames ]] && [[ "$(wc -l $site_path/.hostnames | cut -d ' ' -f1)" != 0 ]]; then
```

Comment:
That's strange. I couldn't reproduce this in the vm, only on the darwin host…but because `cut` was misbehaving since the output is different! I'll keep it though. :)